### DISCUSSION TITLE
Keymaps sometime doesn't work

### DISCUSSION THREAD

**Discussion by batyrmuhadov** [NvChad/NvChad#3082] [Q&A] [answered]
I recently switched from LazyVim to NvChad, and so far, I like it. However, there’s one thing that's bothering me: sometimes keymaps don’t seem to work consistently. For example, when I try to open Telescope by pressing the leader key (space) ff, nothing happens. 🤷‍♂️ This worked perfectly in LazyVim.

P.S. I’m using the same version of Neovim, 0.10.1.

**Comment by siduck** [comment, marked as answer]
do load nvchad mappings. No one is facing this issue except you

**Reply by batyrmuhadov** [reply]
I did, if I didn't it won't work at all, but as I said it doesn't work "SOMETIMES". Maybe I should set timeouts, what do you think can it solve the issue?

**Reply by siduck** [reply]
i have no idea.  it works fine here, ig you are not pressing the mappings fast

https://github.com/user-attachments/assets/d4b00edf-894b-45ed-8afb-6adc364abd9c

**Reply by batyrmuhadov** [reply]
idk why but it seems like vim.schedule gives some delay so I solved it by loading mappings immediatly before configs and plugins. Thanks
